spring-cloud-skipper/pom.xml

```diff
@@ -41,7 +41,7 @@
 		<jmockit.version>1.24</jmockit.version>
 
 		<testcontainers.version>1.17.6</testcontainers.version>
-		<commons-compress.version>1.24.0</commons-compress.version>
+		<commons-compress.version>1.26.0</commons-compress.version>
```

Comment:
Looks like this was already here, but we should not need to override here in skipper as it inherits from one of the above. I am ok leaving it in for this PR (up to you).

Reply:
I'm leaving the build in 2.11.x as is.